### PR TITLE
Lpx updates

### PR DESCRIPTION
**Pull request by DavidArmstrongGSQ**
Updated UAT vocabularies - sampling-method

### CODE REVIEW

File: uat-vocabularies/geological-observation-type.ttl [hunk 1]
```diff
@@ -611,6 +611,15 @@ obstp:younging a skos:Concept ;
     skos:prefLabel "Younging"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/geological-observation-type> .
 
+obstp:lot-adjacent a skos:Concept ;
+    skos:definition "Is the lot-on-plan number used to describe a feature not within the bounds of that parcel, but instead, adjacent to."@en ;
+    rdfs:isDefinedBy <http://linked.data.gov.au/def/geological-observation-type> ;
+    skos:inScheme <http://linked.data.gov.au/def/geological-observation-type> ;
+    skos:notation "LA" ;
+    skos:prefLabel "Lot Adjacent"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/geological-observation-type> .
```
Comment (KellyVance) [inline uat-vocabularies/geological-observation-type.ttl:620]: Need some clarity on the purpose of this.

**Review by KellyVance — CHANGES_REQUESTED** [review]
Few general comments below.
Also please update modified dates.

File: uat-vocabularies/sampling-method.ttl [hunk 2]
```diff
@@ -1041,3 +1041,25 @@ gasmp:underwater_video_rov a skos:Concept ;
...
+
+
+gasmp:underwater_video_rov a skos:Concept ;
+	dcterms:created "2018-01-30T23:26:29Z"^^xsd:dateTime ;
+	dcterms:creator <http://editor.vocabs.ands.org.au/user/GA_Sedgmen.Lara_Admin> ;
+	dcterms:provenance "Geoscience Australia"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/sampling-method> ;
+	skos:definition "A video taken using an underwater camera on a remotely operated vehicle."@en ;
+	skos:prefLabel "underwater video ROV"@en ;
+	skos:broader gasmp:underwater_video .
```
[the remark this answers is not in the document]
Comment (DavidArmstrongGSQ) [reply uat-vocabularies/sampling-method.ttl]: Removed

File: uat-vocabularies/geological-observation-type.ttl (hunk 1 above)
Comment (DavidArmstrongGSQ) [reply uat-vocabularies/geological-observation-type.ttl:620]: @GSQ-AI  are you able to help? I'm not up to date with quarries

File: uat-vocabularies/geo-commodities.ttl [hunk 3]
```diff
@@ -3779,6 +3779,14 @@ cdt:greenhouse-gases a skos:Concept ;
...
     skos:prefLabel "Greenhouse Gases"@en .
 
+cdt:quarry-product a skos:Concept ;
+    skos:broader cdt:sequestered-commodity ;
+    skos:definition "Any geocommodity produced by a quarry of economic value."@en ;
+    rdfs:isDefinedBy <http://linked.data.gov.au/def/geo-commodities> ;
+    skos:inScheme <http://linked.data.gov.au/def/geo-commodities> ;
+    skos:notation "QP" ;
+    skos:prefLabel "Quarry Product"@en .
+
```
Comment (KellyVance) [inline uat-vocabularies/geo-commodities.ttl:3789]: I think this is what is trying to be achieved. A collection for the webform to reference and generate a dropdown list for extractive (Quarry) resources. @DavidArmstrongGSQ ?

```suggestion
cdt:quarry-product a skos:Collection ;
    skos:definition "Any geocommodity produced by a quarry of economic value as applicable to Queensland statisical sales reporting."@en ;
    skos:member cdt:asphalt-aggregate,
        cdt:asphalt-sand,
        cdt:broken-rock,
        cdt:concrete-aggregate,
        cdt:pre-cast-concrete-sand,
        cdt:crushed-gravel,
        cdt:crusher-fines,
        cdt:decorative-crushed-rock,
        cdt:decorative-gravel,
        cdt:packing-sand,
        cdt:manufactured-sand,
        cdt:mortar-sand,
        cdt:other-coarse-aggregate,
        cdt:other-sand,
        cdt:ridge-gravel,
        cdt:plaster-sand,
        cdt:pre-mix-concrete-sand,
        cdt:rail-ballast,
        cdt:roadbase,
        cdt:sealing-aggregate,
        cdt:horticultural-sand,
        cdt:unprocessed-construction-material,
        cdt:unprocessed-gravel ;
    skos:prefLabel "Queensland Quarry Products"@en .

```

**Review by DavidArmstrongGSQ — COMMENTED** [review]
In terms of structural changes they look good to me, and happy for quarry to become a collections. Will leave content to Andrew and Luke

File: uat-vocabularies/geo-commodities.ttl (hunk 3 above)
Comment (DavidArmstrongGSQ) [reply uat-vocabularies/geo-commodities.ttl:3789]: Decided duplicate of and no need to add http://linked.data.gov.au/def/geo-commodities/qld-extractive-sales

File: uat-vocabularies/site-status.ttl [hunk 4]
```diff
@@ -95,7 +102,7 @@ ststs:suspended a skos:Concept ;
     skos:altLabel "Drilled"@en,
         "Retention"@en ;
     skos:broader ststs:inactive ;
-    skos:definition "A project or activity that is temporarily not active, such as during the interval between construction and commencement of production or extraction."@en ;
+    skos:definition "A site such as mine or quarry, project or activity that is temporarily not active, such as during the interval between construction and commencement of production or extraction."@en ;
```
Comment (GSQ-AI) [inline uat-vocabularies/site-status.ttl:105]: such as _a_ mine or quarry

File: uat-vocabularies/site-status.ttl [hunk 5]
```diff
@@ -158,6 +165,13 @@ ststs:rehabilitation a skos:Concept ;
     skos:prefLabel "Rehabilitation"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/site-status> .
 
+ststs:processing a skos:Concept ;
+    skos:broader ststs:active ;
+    skos:definition "A site such as mine or quarry which is processing but not mining raw material throughout the reporting period."@en ;
```
Comment (GSQ-AI) [inline uat-vocabularies/site-status.ttl:170]: such as _a_ mine or quarry

**Review by GSQ-AI — APPROVED** [review]
Minor grammatical errors @105 and 170 can be ignored in favour of deploying to support LPX development. 

The work done is of an appropriate quality and standard. Suitable for merge.

File: uat-vocabularies/site-status.ttl (hunk 5 above)
Comment (DavidArmstrongGSQ) [reply uat-vocabularies/site-status.ttl]: Change and committed

**Pull request merged**